test/ng/compileSpec.js

```diff
@@ -231,11 +231,26 @@ describe('$compile', function() {
 
   describe('svg namespace transcludes', function() {
     // this method assumes some sort of sized SVG element is being inspected.
-    function assertIsValidSvgCircle(elem) {
+    function assertIsValidSvgCircle(elem, possiblyUnsizedParent) {
       expect(isUnknownElement(elem)).toBe(false);
       expect(isSVGElement(elem)).toBe(true);
+
       var box = elem.getBoundingClientRect();
-      expect(box.width === 0 && box.height === 0).toBe(false);
+      var hasWidthOrHeight = (box.width !== 0) || (box.height !== 0);
+
+      if (possiblyUnsizedParent && !hasWidthOrHeight) {
```

Comment:
I think it is better to put an `if (isEdge)` condition on the test and do not run the test there at all than this change. I understand that this is an issue with Edge, but we should not try to test something else because of a browser